### DISCUSSION TITLE
dns google.com 返回异常

### DISCUSSION THREAD

**Discussion by ghost** [kkkgo/PaoPaoDNS#112] [问答和帮助]
![image](https://github.com/kkkgo/PaoPaoDNS/assets/93303847/03c4a29f-8ca8-4bbd-ac70-10e3d6e0535b)
![image](https://github.com/kkkgo/PaoPaoDNS/assets/93303847/8c2be808-40d1-4ac6-b84b-9ca363f2e759)

**Comment by kkkgo** [comment]
建议开ADDINFO然后用dig查看具体来源
